src/GitReportCL/Validation.cs

```diff
@@ -0,0 +1,81 @@
+﻿
+
+namespace GitReportCL
+{
+    using System;
+    using System.Globalization;
+    using System.IO;
+    class Validation
+    {
+            public SetGitReport setGitRep = new SetGitReport();
+            private string DateValidation(string arg)
+            {
+                string[] formats = new string[] { "dd/MM/yyyy", "d/M/yyyy", "MM/dd/yyyy", "M/d/yyyy",
+                "dd.MM.yyyy", "d.M.yyyy", "MM-dd-yyyy", "M-d-yyyy", "dd-MM-yyyy", "d-M-yyyy", "MM.dd.yyyy", "M.d.yyyy" };
+
+                while (DateTime.TryParseExact(arg, formats, setGitRep.cultureInfo, DateTimeStyles.None, out setGitRep.dateSince) == false)        //AdjustToUniversal
+                {
+                    Console.WriteLine("Please enter new date:");
+                    arg = Console.ReadLine();
+                }
+                arg = setGitRep.dateSince.ToString();
+                return arg;
+            }
+            public string PathValidation(string arg)
+            {
+                if (arg == null)
+                {
+                    return PathValidationLoop(arg);
+                }
+                if (arg.Contains("Phoenix"))                     //test           Phoenix
+                {
+                    return PathValidationLoop(arg);
+                }
+                else
+                {
+                    // Console.WriteLine("Your path was not the path to Phoenix repository. Set a new path:");
+                    return PathValidationLoop(Console.ReadLine());
+                }
+            }
+            public string PathValidationLoop(string arg)
+            {
+                while (Directory.Exists(arg) == false)
```

Comment:
* Should use `!`. Instead of `== false`
* Even single lines should be wrapped with `{` and `}`